lib/plausible_web/controllers/api/external_controller.ex

```diff
@@ -167,8 +167,13 @@ defmodule PlausibleWeb.Api.ExternalController do
 
   defp is_bot?(%UAInspector.Result.Bot{}), do: true
 
-  defp is_bot?(%UAInspector.Result{client: %UAInspector.Result.Client{name: "Headless Chrome"}}),
-    do: true
+  defp is_bot?(%UAInspector.Result{client: %UAInspector.Result.Client{name: "Headless Chrome"}}) do
+    if Application.get_env(:plausible, :system_environment) == "staging" do
```

Comment:
I think if we want to support release candidate environment we'd need to check for it as well.

Reply:
Or do we want to treat both as "staging" ? I was thinking about simply reusing the environment devops but I'm pretty sure they set different values for staging and release candidate

Reply:
Yes: https://github.com/Recruitee/k8-infrastructure/blob/master/overlays/rc/apps/careers-analytics/env.txt#L7